### PR TITLE
Update trace search and analytics page

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Updates docs with UI changes, and adds a couple clarifying mp4s

### Motivation
Request from PM, DOCS-2884

### Preview

https://docs-staging.datadoghq.com/kari/docs-2284-trace-search/tracing/trace_search_and_analytics/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_search_and_analytics/_index.md [hunk 1]
```diff
@@ -78,24 +78,21 @@ A valid query in the search bar displays traces that match your search criteria
...
 {{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-4.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}
 
 You can search retained traces in the same way as you do a Live Search. To switch from searching live to retained data, change the time selector to any period of time greater than 15 minutes.
 
-All spans that are indexed by retention filters or legacy App Analytics filters are accessible from search.  These spans are kept by Datadog for 15 days after being indexed by a retention filter.
-
-**Note:** As of October 20, 2020 Tracing without Limits replaced App Analytics as a more flexible way to ingest 100% of your traces and retain the ones important to your business.
-
-{{< img src="tracing/live_search/HistoricalSearch2.mp4" alt="Historical Search" video=true >}}
+All spans that are indexed by retention filters are accessible from search.  These spans are kept by Datadog for 15 days after being indexed by a retention filter.
```
Comment (sarina-dd) [inline content/en/tracing/trace_search_and_analytics/_index.md:93]:
```suggestion
All spans that are indexed by retention filters are accessible from search. These spans are kept by Datadog for 15 days after being indexed by a retention filter.
```

File: content/en/tracing/trace_search_and_analytics/_index.md [hunk 2]
```diff
@@ -17,23 +17,23 @@ description: "Trace Search and Analytics"
 
 The Datadog app shows a **Live** indicator next to the time selector whenever you are in Live mode:
 
-{{< img src="tracing/live_search/LiveSearch.png" alt="Live Search Indicator" >}}
+{{< img src="tracing/live_search/live-search.png" alt="Live Search Indicator"   style="width:80%;" >}}
```
Comment (sarina-dd) [inline content/en/tracing/trace_search_and_analytics/_index.md:20]:
```suggestion
{{< img src="tracing/live_search/live-search.png" alt="Live Search Indicator" style="width:80%;" >}}
```

**Review by sarina-dd — APPROVED** [review]
LGTM (left small spacing nits + comment about video sizing)

**Comment by kayayarai** [comment]
Holding for roll-out. This is approved by the PM and ready for merge as soon as Engineering requests it in documentation channel.

**Pull request merged**